internal/cli/login/oauth.go

```diff
@@ -109,13 +117,19 @@ func oauth2RefreshTokenFlow(refreshToken string, getClient GetClientFunc) (AuthI
 
 	// callback url is not used in refresh token flow, but has to be set, otherwise osincli fails
 	// the value should not matter
-	client, err := getClient("http://127.0.0.1")
+	client, clientId, err := getClient("http://127.0.0.1")
```

Comment:
prefer using localhost and probably something like :
http://localhost:8080/callback (or some other port)